src/features/common/Layout/Navbar/getNavBarIcon.tsx

```diff
@@ -44,7 +58,13 @@ function GetNavBarIcon({ mainKey, router, item }: Props): ReactElement {
         }`}
       >
         {router.pathname === item.onclick || router.pathname === '/[p]' ? (
-          <DonateSelected color={themeProperties.primaryColor} />
+          tenantName === 'planet' ? (
+            <HomeLogoSelected color={themeProperties.primaryColor} />
+          ) : (
+            <DonateSelected color={themeProperties.primaryColor} />
+          )
+        ) : tenantName === 'planet' ? (
+          <HomeLogo />
```

Comment:
1. Nested ternary conditions are difficult to read later on. Can this be simplified (perhaps extract some of this logic into a separate function).
2. Pass color as a prop for HomeLogo similar to how is done for other icons.